book/tutorials/Data_access/NSIDC_resources.md

```diff
@@ -0,0 +1,79 @@
+# Exploring NSIDC DAAC resources
+
+## Learning Objectives
+
+Explore various resources for learning about and accessing ICESat-2, SnowEx, and other NASA Earthdata.
+
+___
+
+Reach out to us with data discovery and access questions!  Real people read the emails sent to nsidc@nsidc.org.  We are here to help make sure you get the data you need for your analysis.
+___
+
+## NASA National Snow and Ice Data Center Distributed Active Archive Center (NSIDC DAAC)
+
+[The National Snow and Ice Data Center](https://nsidc.org) provides over 1100 data sets covering the Earth's cryosphere and more, all of which are available to the public free of charge. Beyond providing these data, NSIDC creates tools for data access, supports data users, performs scientific research, and educates the public about the cryosphere. 
+
+<figure>
+<center>
+   <img width="80%" src='./images/DAAC_map_with_ECS.jpg'/>
+</center>
+</figure>
```

Comment:
This are not rendering on CryoCloud, could you please try this syntax https://mystmd.org/guide/figures#image-directive

Reply:
Will do!